src/services/userService.js

```diff
@@ -1,23 +1,27 @@
-import { user } from './mocks/userData';
-
 export function logIn({ login = '', password = '' }) {
 	return new Promise((resolve, reject) => {
-		if(login === user.login && password === user.password) {
+		const parsedUserData = JSON.parse(localStorage.getItem('user'));
```

Comment:
JSON.parse should be always wrapped by try catch.

Reply:
done